src/scss/components/card.scss

```
@@ -317,30 +349,50 @@
     }
   }
 
-  .media--small.media--circle {
-    max-width: 11.375rem;
+  .media--widescreen .media-date {
+    width: unset;
   }
 
-  .media--small.media--square {
+  .media--xx-small {
+    max-width: 5.375rem;
+
+    &.media--widescreen {
+      @include container-query() {
+        max-width: 11.375rem;
+      }
+    }
+  }
+
+  .media--x-small {
+    max-width: 11.375rem;
+
     @include container-query() {
       max-width: 11.375rem;
     }
   }
 
-  .media--small.media--widescreen {
+  .media-small {
```

```suggestion
  .media--small {
```
Probably should be this?

---

Yes! Thank you